ramls/request.json

```diff
@@ -170,6 +170,11 @@
       "type": "object",
       "description": "Tags",
       "$ref": "raml-util/schemas/tags.schema"
+    },
```

Comment:
This new property requires a minor version increment in the interface version (both in the RAML and module descriptor)

Reply:
done